src/controller/config/controller.go

```diff
@@ -174,6 +178,10 @@ func ConvertForGet(ctx context.Context, cfg map[string]interface{}, internal boo
 	return result, nil
 }
 
-func authModeCanBeModified(ctx context.Context) (bool, error) {
-	return dao.AuthModeCanBeModified(ctx)
+func (c *controller) authModeCanBeModified(ctx context.Context) (bool, error) {
+	users, err := c.userManager.List(ctx, &q.Query{})
+	if err != nil {
+		return false, err
+	}
+	return len(users) == 0, nil
```

Comment:
The system will always have a admin user, is it correct?

Reply:
userManager already filter it out